src/Elastic.Markdown/Myst/Renderers/HtmxLinkInlineRenderer.cs

```diff
@@ -25,7 +32,8 @@ protected override void Write(HtmlRenderer renderer, LinkInline link)
 				return;
 			}
 
-			var url = link.GetDynamicUrl != null ? link.GetDynamicUrl() : link.Url;
+			var url = link.GetDynamicUrl?.Invoke() ?? link.Url;
```

Comment:
I'm not sure if we want this here. 

we only want to set it in the `hx-get` attribute.

otherwise, clicking on the link will contain the querystring